python/lsst/pipe/tasks/cosmicRayPostDiff.py

```diff
@@ -0,0 +1,263 @@
+
+    @section pipe_tasks_cosmicRayPostDiff_Initialize  Task initialisation
+
+    @copydoc \_\_init\_\_
+
+    @section pipe_tasks_cosmicRayPostDiff_IO  Invoking the Task
+
+    Call all the `run` method:
+
+    @section pipe_tasks_cosmicRayPostDiff_Config  Configuration parameters
+
+    See @ref CosmicRayPostDiffConfig
+
+    @section pipe_tasks_cosmicRayPostDiff_Debug  Debug variables
+
+    The @link lsst.pipe.base.cmdLineTask.CmdLineTask command line task@endlink interface supports a flag
+    `--debug` to import `debug.py` from your `$PYTHONPATH`; see @ref baseDebug for more about `debug.py`.
+
+    CosmicRayPostDiffTask has a debug dictionary with the following keys:
+    <dl>
+    <dt>frame
+    <dd>int: if specified, the frame of first debug image displayed (defaults to 1)
+    <dt>repair
+    <dd>bool; if True display image and sources after final repair
+
+    </dl>
+
+    For example, put something like:
+    @code{.py}
+        import lsstDebug
+        def DebugInfo(name):
+            di = lsstDebug.getInfo(name)  # N.b. lsstDebug.Info(name) would call us recursively
+            if name == "lsst.pipe.tasks.cosmicRayPostDiff":
+                di.display = dict(
+                    repair = True,
+                )
+
+            return di
+
+        lsstDebug.Info = DebugInfo
+    @endcode
+    into your `debug.py` file and run `calibrateTask.py` with the `--debug` flag.
+
+    Some subtasks may have their own debug variables; see individual Task documentation.
+    """
+
+    ConfigClass = CosmicRayPostDiffConfig
+    _DefaultName = "cosmicRayPostDiff"
+    RunnerClass = pipeBase.ButlerInitializedTaskRunner
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+        if 'exposureIdInfo' not in inputs.keys():
+            inputs['exposureIdInfo'] = ExposureIdInfo.fromDataId(butlerQC.quantum.dataId, "visit_detector")
+        outputs = self.run(**inputs)
+        butlerQC.put(outputs, outputRefs)
+
+    def __init__(self, butler=None, refObjLoader=None, schema=None, **kwargs):
+        """!Construct a CosmicRayPostDiffTask
+
+        @param[in] butler  A butler object is passed to the refObjLoader constructor in case
+            it is needed to load catalogs.  May be None if a catalog-based star selector is
+            not used, if the reference object loader constructor does not require a butler,
+            or if a reference object loader is passed directly via the refObjLoader argument.
+        @param[in] refObjLoader  An instance of LoadReferenceObjectsTasks that supplies an
+            external reference catalog to a catalog-based star selector.  May be None if a
+            catalog star selector is not used or the loader can be constructed from the
+            butler argument.
+        @param[in,out] schema  initial schema (an lsst.afw.table.SourceTable), or None
+        @param[in,out] kwargs  other keyword arguments for lsst.pipe.base.CmdLineTask
```

Comment:
Update the docstring format to pydoc style, and remove references to unused parameters.